### PR TITLE
Data persistence(working model)

### PR DESCRIPTION
**Pull request by rbolline**

### CODE REVIEW

File: src/App.jsx [hunk 1]
```diff
@@ -34,14 +36,37 @@ function DocumentEditor({ document, updateDocument }) {
   );
 }
 
+function useLocalStorage(key, initialValue) {
```
Comment (nahtnam) [inline src/App.jsx]: move this to inside the App function

File: src/App.jsx [hunk 2]
```diff
@@ -34,14 +36,37 @@ function DocumentEditor({ document, updateDocument }) {
...
+  return [storedValue, setValue];
+}
+
 
 function App() {
-  const [docStorage, setDocStorage] = useState(['hi', 'hello']);
+  //key == value in local storage
+  const [docStorage, setDocStorage] = useLocalStorage('doclist', ['hi', 'hello']);
   const [selectedDocument, selectDocument] = useState(0);
+  ls("wow", "w");
```
Comment (nahtnam) [inline src/App.jsx]: what is this?

**Review by nahtnam — CHANGES_REQUESTED** [review]

File: src/App.jsx (hunk 2 above)
Comment (rbolline) [reply src/App.jsx]: removed

File: src/App.jsx (hunk 1 above)
Comment (rbolline) [reply src/App.jsx]: done

File: src/App.jsx [hunk 3]
```diff
@@ -5,6 +5,8 @@ import classnames from 'classnames';
 import 'bulma/css/bulma.css';
 import './styles/App.css';
 
+const ls = require('local-storage');
```
Comment (nahtnam) [inline src/App.jsx]: use import syntax: `import ls from 'local-storage';`

Comment (nahtnam) [reply src/App.jsx]: and move it to the top

**Review by nahtnam — CHANGES_REQUESTED** [review]

File: src/App.jsx (hunk 3 above)
Comment (rbolline) [reply src/App.jsx]: done

**Review by nahtnam — APPROVED** [review]

**Pull request merged**